src/main/java/org/candlepin/subscriptions/ApplicationConfiguration.java

```diff
@@ -84,6 +85,7 @@
   // NOTE(khowell): actually not needed in marketplace worker
   RhsmSubscriptionsDataSourceConfiguration.class,
 })
+@ComponentScan("org.candlepin.subscriptions.util")
```

Comment:
I don't object to this but it's a little different from what we're doing elsewhere where we create a XYZConfiguration class in the XYZ package with the `ComponentScan` on it.  If you feel strongly about doing it the way you have it, I'm fine with that, but I thought it might be nice to remain consistent.

Reply:
I can do that. Hopefully we'll be out of this mess soon via refactoring :-)